src/server/endpoints/covidcast_utils/model.py

```diff
@@ -241,18 +241,18 @@ def get_related_signals(signal: DataSignal) -> List[DataSignal]:
     return [s for s in data_signals if s != signal and s.signal_basename == signal.signal_basename]
 
 
-def count_signal_time_types(source_signals: List[SourceSignalPair]) -> Tuple[int, int]:
+def count_signal_time_types(source_signals: List[SourceSignalSet]) -> Tuple[int, int]:
     """
     count the number of signals in this query for each time type
     @returns daily counts, weekly counts
     """
     weekly = 0
     daily = 0
-    for pair in source_signals:
-        if pair.signal == True:
+    for set in source_signals:
```

Comment:
`set` is a [builtin](https://docs.python.org/3/library/functions.html#built-in-funcs) and we should avoid using it outside of that context.  maybe use something like `ssset` or `ss_set` instead of just `set` here?